libdevcore/IndentedWriter.h

```diff
@@ -34,7 +34,7 @@ DEV_SIMPLE_EXCEPTION(IndentedWriterError);
 class IndentedWriter
 {
 public:
-	explicit IndentedWriter(): m_lines(std::vector<Line>{{std::string(), 0}}) {}
+	IndentedWriter() = default;
```

Comment:
I think if `ClassName() = default;` is the only constructor, we should rather remove it completely.